ps_googleanalytics.php

```diff
@@ -167,7 +177,18 @@ public function hookActionProductCancel($params)
     }
 
     /**
-     * hook save cart event to implement addtocart and remove from cart functionality
+     * Hook called after order status change, used to "refund" order after cancelling it
+     */
+    public function hookActionOrderStatusPostUpdate($params)
+    {
+        $hook = new PrestaShop\Module\Ps_Googleanalytics\Hooks\HookActionOrderStatusPostUpdate($this, $this->context);
```

Comment:
What about an import here? :)

Reply:
If you want to improve the code on `dev` be my guest :) I will not have the time (right now) to do more than opening the PR to merge `dev` into `master`

Reply:
I think we shouldn't review the release pull request, otherwise, why not review the 1.7.7.4 release :trollface: 
We should review and improve code when it targets the dev branch.